modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,111 @@
 // Module included in the following assemblies:
-// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+// | virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 [id='virt-creating-configmap_{context}']
-= Creating a custom ConfigMap for importing a virtual machine
+= Creating a custom ConfigMap
 
-If the operating system of the Red Hat Virtualization (RHV) virtual machine does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap to map the RHV operating system to a common {VirtProductName} template. The template is used to create the new virtual machine.
+You can create a custom ConfigMap for a Red Hat Virtualization (RHV) virtual machine if its operating system does not have a corresponding {product-title} template.
 
 [NOTE]
 ====
-In the current release, you must create a Windows template and a custom ConfigMap to import a RHV VM with a Windows operating system. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ#1856654] for details.
+If you are importing a Windows virtual machine, you must also create a Windows template. link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[(*BZ#1856654*)]
 ====
 
 .Procedure
 
-. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` as in the following example:
+. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>`. The operating system appears in the `<os>` section of the XML output, as shown in the following example:
 +
 [source,xml]
 ----
 <os>
 ...
-<type>windows_2016x64</type>
+<type>rhel_8x64</type>
 </os>
 ----
 
-. Obtain a list of the common {VirtProductName} operating system templates by running the following command on your {product-title} cluster:
+. To import a Linux VM:
```

Comment:
There is no point in checking if it is Windows because a CNV template has to be created. Only Linux systems will have a CNV template, so only Linux VMs are relevant. That's why I created a Linux procedure.

Reply:
As we discussed documenting the Windows W/A is temporary, and here we need to doc both Windows and Linux.
Soon the W/A will be removed.